frontend/routes/viewer/components/risks/components/riskDetails/riskDetailsForm.component.tsx

```diff
@@ -170,71 +259,49 @@ class RiskDetailsFormComponent extends React.PureComponent<IProps, IState> {
 				)}
 
 				<FieldsRow container alignItems="center" justify="space-between">
-					<StyledFormControl>
-						<InputLabel shrink={true} htmlFor="assigned_roles">Risk owner</InputLabel>
-						<Field name="assigned_roles" render={({ field }) => (
-							<CellSelect
-								{...field}
-								items={this.props.jobs}
-								inputId="assigned_roles"
-								disabled={!this.props.canUpdateRisk}
-							/>
-						)} />
-					</StyledFormControl>
-
-					<StyledFormControl>
-						<InputLabel shrink={true} htmlFor="category">Category</InputLabel>
-						<Field name="category" render={({ field }) => (
-							<CellSelect
-								{...field}
-								items={RISK_CATEGORIES}
-								inputId="category"
-								disabled={!this.props.canUpdateRisk}
-							/>
-						)} />
-					</StyledFormControl>
-				</FieldsRow>
+					<FieldsContainer>
+						<StyledFormControl>
+							<InputLabel shrink={true} htmlFor="residual_likelihood">Mitigated Likelihood</InputLabel>
+							<Field name="residual_likelihood" render={({ field }) => (
+								<CellSelect
+									{...field}
+									items={RISK_LIKELIHOODS}
+									inputId="residual_likelihood"
+									disabled={!this.props.canUpdateRisk}
+								/>
+							)} />
+						</StyledFormControl>
 
-				<FieldsRow container alignItems="center" justify="space-between">
-					<StyledFormControl>
-						<InputLabel shrink={true} htmlFor="likelihood">Risk Likelihood</InputLabel>
-						<Field name="likelihood" render={({ field }) => (
-							<CellSelect
-								{...field}
-								items={RISK_LIKELIHOODS}
-								inputId="likelihood"
-								disabled={!this.props.canUpdateRisk}
-							/>
-						)} />
-					</StyledFormControl>
+						<StyledFormControl>
```

Comment:
I prefer leaving them as tabs, as we'd have to touch every file in the repo if we change them to spaces, also different people prefer different indent size so leaving them as tabs gives that flexibility.

According to [this](https://stackoverflow.com/a/33831598) we can add an editorconfig file to set this and it will be respected in the github viewer. so we could do that if you like.

Reply:
Ok! ;)